### DISCUSSION TITLE
File name fix

### DISCUSSION THREAD

**Discussion by AFumi39** [lavinir/hassio-onedrive-backup#110] [Ideas]
It would be great if was possibile to fix "external" backup file name before trying to upload it (otherwise it tries to upload it indefinitely). For example if a backup have a time in name with ":" char (fo example "Partial_Backup_2023.9.2_2023-09-25 01:00") it fails to upload to OneDrive due to its name.
It would be great if it could "sanitize" filenames with (for example) "Partial_Backup_2023.9.2_2023-09-25 01_00".

Thank you

**Comment by lavinir** [comment]
Created Issue #111
